contracts/modules/TransferManager/CountTransferManagerFactory.sol

```diff
@@ -43,41 +39,9 @@ contract CountTransferManagerFactory is ModuleFactory {
      * @return address Contract address of the Module
      */
     function deploy(bytes calldata _data) external returns(address) {
-        address polyToken = _takeFee();
-        address countTransferManager = address(new CountTransferManagerProxy(msg.sender, polyToken, logicContract));
-        require(Util.getSig(_data) == IBoot(countTransferManager).getInitFunction(), "Provided data is not valid");
-        bool success;
-        /*solium-disable-next-line security/no-low-level-calls*/
-        (success, ) = countTransferManager.call(_data);
-        require(success, "Unsuccessful call");
-        /*solium-disable-next-line security/no-block-members*/
-        emit GenerateModuleFromFactory(address(countTransferManager), getName(), address(this), msg.sender, getSetupCost(), getSetupCostInPoly(), now);
-        return address(countTransferManager);
-    }
-
-    /**
-     * @notice Type of the Module factory
-     */
-    function getTypes() external view returns(uint8[] memory) {
-        uint8[] memory res = new uint8[](1);
-        res[0] = 2;
-        return res;
-    }
-
-    /**
-     * @notice Returns the instructions associated with the module
-     */
-    function getInstructions() external view returns(string memory) {
-        return "Allows an issuer to restrict the total number of non-zero token holders";
+        address countTransferManager = address(new CountTransferManagerProxy(logicContracts[latestVersion].version, msg.sender, IPolymathRegistry(polymathRegistry).getAddress("PolyToken"), logicContracts[latestVersion].logicContract));
+        _initializeModule(countTransferManager, _data);
+        return countTransferManager;
     }
 
-    /**
-     * @notice Get the tags related to the module factory
-     */
-    function getTags() external view returns(bytes32[] memory) {
```

Comment:
In every other module we are using the `tags()` function and `types()` function but here not. and we are not storing this info into the state variables of the modules but here we are storing them. 

we have the function `changeTags()` in `ModuleFactory.sol` which changes the tags of the factory but we call the `tags()` function it always returns the same value as it is not returning from the `tagsData` array (overriding the `ModuleFactory` functions). The same problem with `types()`

Reply:
This should all be correct in the PR - oddly you seem to see an old version of the code!

Could you check again this point - it should be fixed in the commit from last night.